package.json

```diff
@@ -196,6 +196,7 @@
       "dist/v0/*-?.?.js",
       "dist/*.mjs",
       "dist/v0/*-?.?.mjs"
-    ]
+    ],
+    "trackFormat": ["brotli"]
```

Comment:
this filters out the reporting to just brotli correct?

Reply:
This option specifies which formats (compression formats) to track. In the case of AMP, the bundlesize check only cares about `brotli`.

Reply:
I just noticed that `yarn add` and `prettier` [do not agree](https://travis-ci.org/github/ampproject/amphtml/jobs/678816324#L406-L411) on how this setting should be formatted inside `package.json`. I've filed https://github.com/ampproject/filesize/issues/83 to request that the suggestion in https://github.com/ampproject/amphtml/pull/26907#discussion_r383399093 (a separate config file for `filesize`) be implemented.